gitalias.txt

```diff
@@ -221,6 +221,12 @@
   # merge but without autocommit, and with a commit even if the merge resolved as a fast-forward.
   me = merge --no-commit --no-ff
 
+  # merge abort - cancel the merging process
```

Comment:
May I suggest to put these aliases above `me` so that the section is ordered alphabetically?

Reply:
Reordered

Reply:
Nice, thanks!